serverlessworkflow/modules/ROOT/pages/integrations/custom-functions-knative.adoc

```diff
@@ -0,0 +1,335 @@
+= Invoking Knative services from {context} custom functions
+:compat-mode!:
+// Metadata:
+:description: Explain what is and how the Knative service discovery works
+:keywords: kogito, workflow, quarkus, serverless, service-discovery, knative
+
+This document describes how to call Knative services using {context} custom functions. The procedure described in this document is based on the link:{kogito_sw_examples_url}/serverless-workflow-custom-function-knative[`serverless-workflow-custom-function-knative`] example application.
+
+The testing procedure in this document is based on the serverless-workflow-examples/serverless-workflow-custom-function-knative example application. You can access this example application in the link:{kogito_sw_examples_url}/serverless-workflow-custom-function-knative[Kogito Examples] GitHub repository.
+
+[[enabling-knative-custom-functions]]
+== Enabing Knative {context} custom functions
+
+You can enable Knative {context} custom functions in your project.
+
+.Prerequisites
+
+include::../cloud/common/_prerequisites.adoc[]
```

Comment:
Missing attributes:
```
{environment_prereq}

kubectl {kubectl_prereq}
```

Reply:
also do we really need all those prerequisites ?
In the procedure, you just mention to add the new dependency but you never show how to do it with Maven, quarkus CLI or kn CLI ...

Reply:
Yeah, we don't need them.
Removed.